scanners/nessus/nessus_none.py

```diff
@@ -12,6 +15,7 @@
 from configmodel import RapidastConfigModel
 from scanners import RapidastScanner
 from scanners import State
+from scanners.generic.tools.convert_nessus_csv_to_sarif import convert_csv_to_sarif
```

Comment:
Working on this now. Reviewing the directory structure a bit more, does it make more sense to move the convert_nessus_csv_to_sarif file to something like scanners/nessus/tools instead of the generic directory where it is currently located as well?

Reply:
Tests added. Went ahead and did that move, which can obviously be undone.

The linter is unhappy, but I  believe it is only the Nessus CSV file formatting it takes issue with

Reply:
Gah that's annoying. Try adding this patch to exclude CSV files from this check:

```
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 8408bc9..878ab3e 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -4,6 +4,7 @@ repos:
     hooks:
       - id: trailing-whitespace
         language_version: python3
+        exclude: \.csv$
       - id: end-of-file-fixer
       - id: debug-statements
         language_version: python3
```

Reply:
That worked! Thanks!